source/index.html.md.erb

```diff
@@ -3,22 +3,12 @@ title: "GOV.UK Platform as a Service Technical Documentation"
 ---
 
 <%= partial 'documentation/index' %>
-<%= partial 'documentation/overview/benefits' %>
-<%= partial 'documentation/overview/technology' %>
-<%= partial 'documentation/overview/pricing' %>
-<%= partial 'documentation/getting_started/before_you_start' %>
-<%= partial 'documentation/getting_started/quick_setup_guide' %>
-<%= partial 'documentation/getting_started/passwords' %>
-<%= partial 'documentation/getting_started/system' %>
-<%= partial 'documentation/getting_started/limitations' %>
-<%= partial 'documentation/getting_started/privacy' %>
 <%= partial 'documentation/orgs_spaces_users/index' %>
 <%= partial 'documentation/deploying_apps/index' %>
 <%= partial 'documentation/deploying_apps/buildpacks' %>
 <%= partial 'documentation/deploying_apps/excluding_files' %>
 <%= partial 'documentation/deploying_apps/env_variables' %>
 <%= partial 'documentation/deploying_apps/names_routes_domains' %>
-<%= partial 'documentation/deploying_apps/deploying_static_sites' %>
```

Comment:
Content best practice is to put any and all quick start or get started content in one place so that users do not have to look in multiple sections. The deploy a static site content comes under this category, therefore I believe it should live in the Get started section. I'll link to the getting started section from the deploying apps section as a compromise.